### ISSUE TITLE
Requires accessibility/inclusivity section

### ISSUE THREAD

**Issue by aral** [elementary/hig#49]
### Problem

The Human Interface Guidelines (HIG) does not have a section on accessibility/inclusivity.

In addition to having its own section, every example provided throughout the HIG should have relevant accessibility notes and links to relevant developer documentation included with examples of how to accessibly implement the said patterns and components.

### Proposal

This will initially require a commitment from elementary OS to making accessibility a core tenet and a cross-cutting concern. While this issue can be used to track this, the actual implementation should likely involve a discussion.

### Prior Art (Optional)

See Apple’s Accessibiltiy section in their Human Interface Guidelines for a best-in-breed example:

https://developer.apple.com/design/human-interface-guidelines/accessibility/overview/introduction/

**Comment by Suzie97** [comment]
Accessibility guidelines would be useful for developers. I've talked to friends about accessibility, and they say they are interested but don't necessarily know how to make their products more accessible. A good set of guidelines that developers can easily follow would be a great plus and ensure more accessible apps in AppCenter.
